our-work/collaborations/nlp.yml

```diff
@@ -0,0 +1,18 @@
+title: NLP Syllabi Project 
```

Comment:
Collaborations are typically a description of a group and how we collaborate with that group. So in this case it Ould just be the Brown University Library (or the PI).  This `yaml` is a little closer to what we do on `Software`, which is more 1:1 t projects. Since this actually has source code and a website I would move it there.

Use a more explicit name for the `yaml` file maybe `nlp-syllabi`